doc/vim-plugin.md

```diff
@@ -69,6 +69,13 @@ Config files
 Phpactor works best with Composer - but much functionality including
 auto-completion can still work (sometimes slowly depending on project size).
 
+### Troubleshooting
+
+**E117: Unknown function: phpactor#Status**:
+
+When using vim-plug, you will need to open a PHP file before Phpactor is registered.
+For best results, `cd` to a project that uses Git and Composer before starting Vim.
```

Comment:
I guess this is two peices of advice? i.e. one for the error, and the other is for "best results".

What about:

```
**E117: Unknown function: phpactor#Status**:

The `vim-plug` (and similar) package managers will lazily load plugins only when they are relevant (i.e. when you open a PHP file). if you receive the above error, then you should open a PHP file in VIM before issuing the command.
```

Not sure if we already have a hint about starting VIM in the project root?

Reply:
Or maybe it should be mentioned before `Now issue the following command ...` above? As we are assuming already that the user is using `vim-plug`?

Reply:
@dantleech Thanks for the feedback.

I've reworded the error explanation as you suggested. I'll be out for the rest of the day but let me know your thoughts and I can make any changes (especially where composer/git are concerned) later on.

#### Composer/git bits

I've moved this to its own subheader to keep it separate from the actual 'E117' error. I think it could still be useful though, so rather than remove it entirely I've reworded to be less of a "tip" and more of an explanation of *why* it couldn't be detected.

While the readme and Standalone pages say *"Phpactor works better with Git and Composer"*, I couldn't find anywhere that specifically mentions starting from the project directory.